src/sql/statement.rs

```diff
@@ -709,24 +710,43 @@ fn kafka_sink_builder(
     let encoder = Encoder::new(desc);
     let value_schema = encoder.writer_schema().canonical_form();
 
+    let mut with_options = normalize::with_options(&with_options);
+
+    // Use the user supplied value for replication factor, or default to 3
+    let replication_factor = match with_options.remove("replication_factor") {
```

Comment:
I think it would be better to require users to set this, rather than using a default. Also interested in what @benesch thinks.

Reply:
I have a small preference for allowing a default that will be sane for >90% of kafka deployments but I'll defer to you and nikhil

Reply:
will it still work for people whose kafka installations have fewer than 3 replicas available?

Reply:
If testdrive is any indication, it is a hard error if the replication factor is greater than the number of available replicas. I don't have a strong opinion on this. All the options seem bad.

Reply:
could we use `describe_configs` to read in the `default.replication_factor` setting if there is one and if there is - use that or force the user to specify?

Reply:
for the purposes of this PR I'll move the default replication factor to 1 -- this can let users move forward on master and we haven't made a breaking syntax change? We later decide to move to a smarter default with `describe_configs` or to make the replication factor mandatory?